### PR TITLE
[NO-NSTK] PSO v6.1.0 release

### PR DESCRIPTION
**Pull request by david-enli**
PSO 6.1.0 release:
Provided upgrade path from PSO 5.x to PSO 6.x

### CODE REVIEW

**Comment by sdodsley** [comment]
Is the upgrade tool the only thing in this release???

If so, I'm not convinced that it justifies a full minor release for that based on versioning semantics rules

**Comment by dsupure** [comment]
@sdodsley : Yes, this is the only customer-facing feature. `6.1.0` is the version we promise customers to have the upgrade tool ready, right?

**Comment by sdodsley** [comment]
@dsupure so whilst I agree that 6.1 was the release promised to include the upgrade tool, it's a bit disingenuous to say this is a minor release as there are no new features.
Now if there was a new feature, like the pre-node config, that would definitely justify the increment of the dot-y.
I feel that, given the EOS of PSO, 6.1 should be our last feature release and we should only be doing bug (dot-z) releases until the integration in PX happens.
What is the actual delay on the pre-node config feature?

**Comment by dsupure** [comment]
@sdodsley the upgrade tool is the new feature for this release. So I think 6.1.0 is OK. In addition, 6.1.0 won't be the last release. We will still have a few minor/patch releases in 1H21 to help to stabilize PSO and translate it to the low maintenance status in 2H21.  

pre-node config is not ready to merge given what I saw in the tests. I don't want it messed up `6.1.0`. Let's discuss it in slack.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v6.1.0 (02/24/2021)
+
+#### Enhancements:
+
+* Provided upgrade path from PSO 5.x to PSO 6.x
```
Comment (dsupure) [inline CHANGELOG.md:7]: Can you provide the link to the upgrade doc?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v6.1.0 (02/24/2021)
+
+#### Enhancements:
```
Comment (dsupure) [inline CHANGELOG.md:5]: Please change `Enhancements` to `Features`

Comment (sdodsley) [reply CHANGELOG.md:5]: 👍

**Review by sdodsley — APPROVED** [review]

**Review by pure-jliao — APPROVED** [review]
squash commits please.

**Pull request merged**